addons/l10n_ar/models/account_move_line.py

```diff
@@ -10,14 +10,15 @@ def _l10n_ar_prices_and_taxes(self):
         self.ensure_one()
         invoice = self.move_id
         included_taxes = self.tax_ids.filtered('tax_group_id.l10n_ar_vat_afip_code') if self.move_id._l10n_ar_include_vat() else False
+        price_digits = 10**self.env['decimal.precision'].precision_get('Product Price')
         if not included_taxes:
             price_unit = self.tax_ids.with_context(round=False).compute_all(
-                self.price_unit, invoice.currency_id, 1.0, self.product_id, invoice.partner_id)
-            price_unit = price_unit['total_excluded']
+                self.price_unit * price_digits, invoice.currency_id, 1.0, self.product_id, invoice.partner_id)
+            price_unit = price_unit['total_excluded'] / price_digits
             price_subtotal = self.price_subtotal
         else:
             price_unit = included_taxes.compute_all(
-                self.price_unit, invoice.currency_id, 1.0, self.product_id, invoice.partner_id)['total_included']
+                self.price_unit * price_digits, invoice.currency_id, 1.0, self.product_id, invoice.partner_id)['total_included'] / price_digits
```

Comment:
From my understanding, this call is completely useless: The taxes are already price-included and the quantity is 1.0 so getting the `total_included` at the end will give you the original `price_unit` I think... :thinking:

Reply:
@smetl Also the same approach is here https://github.com/odoo/odoo/blob/13.0/addons/l10n_latam_invoice_document/models/account_move_line.py#L44

Reply:
Hello @pablohmontenegro , 
Thanks for the explanation. Unfortunately, the fact that we did it this way in 13.0 is not sufficient to justify a coding choice made today... 
I would suggest digging the info that @smetl mentioned to find a way to solve the problem at its root.

Reply:
Hello @jco-odoo @vbe-odoo in the example I mentioned in this pr the taxes are included. Because of that I think we used compute_all on 13 and we are still using that complexity. Maybe @zaoral can you help me with the explanation?